ui/app/src/components/LegacyIFrame/LegacyIFrame.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2007-2020 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, { IframeHTMLAttributes } from 'react';
+import { createStyles, makeStyles } from '@material-ui/core/styles';
+import { useEnv } from '../../utils/hooks';
+
+export interface LegacyIFrameBaseProps {
+  iframeProps?: IframeHTMLAttributes<any>;
+}
+
+interface LegacyIFrameProps extends LegacyIFrameBaseProps {
+  path: string;
+  title?: string;
+}
```

Comment:
Make a single interface. In the sub-components, use the the path to the single prop you need (i.e. LegacyIFrameProps['iframeProps']).